decidim-initiatives/spec/shared/create_initiative_example.rb

```diff
@@ -144,53 +110,6 @@
           expect(initiative.signature_end_date).to be_nil
         end
       end
-
-      context "when the initiative type enables custom signature end date" do
```

Comment:
Any reason why we're removing this example?

Reply:
This is being removed because the those particular steps are just for CreateInitiative command which is being used only for the first step in the wizard. After that, the UpdateInitiative command is being used.

Reply:
Fixed in 12f9929a